dwb_local_planner/CMakeLists.txt

```diff
@@ -45,21 +45,22 @@ add_dependencies(debug_dwb_local_planner ${catkin_EXPORTED_TARGETS})
 add_library(trajectory_utils src/trajectory_utils.cpp)
 target_link_libraries(trajectory_utils ${catkin_LIBRARIES})
 
-add_executable(planner_node src/planner_node.cpp)
-target_link_libraries(planner_node ${catkin_LIBRARIES} debug_dwb_local_planner)
-add_dependencies(planner_node ${catkin_EXPORTED_TARGETS})
+add_executable(${PROJECT_NAME}_planner_node src/planner_node.cpp)
```

Comment:
Similarly, can you change this to `local_planner_node` and update `dwb_local_planner/launch/plan_node.launch`?

Reply:
See above.